new_post.php

```diff
@@ -1,43 +1,25 @@
-	<form method="post">
+<form method="post">
 
-		<p><strong><label for ="title">Title:</label></strong><input type="text" name="title" id="title" size="40"/></p>
+	<p><strong><label for ="title">Title:</label></strong><input type="text" name="title" id="title" size="40"/></p>
 
-		<p><textarea cols="80" rows="20" name="entry" id="entry"></textarea></p>
+	<p><textarea cols="80" rows="20" name="entry" id="entry"></textarea></p>
 
-		<p>
-		<p><strong><label for ="changebox">Password protect?:</label></strong><input type="checkbox" name="password" id="password" value="1"/></p>
-			<input type="submit" name="submit" id="submit" value="Submit"/>
-		</p>
-	</form>
-		<a href="./index.php">Home</a>
+	<p><strong><label for ="changebox">Password protect?:</label></strong><input type="checkbox" name="password" id="password" value="1"/></p>
+
+	<p><input type="submit" name="submit" id="submit" value="Submit"></p>
+</form>
+<a href="/index.php">Home</a>
 <?php
+	include('header.inc.php');
 	if(isset($_POST['submit']))
 	{
 		$title = htmlspecialchars(strip_tags($_POST['title']));
 		$entry = $_POST['entry'];
 		$password = isset($_POST['password']) ? htmlspecialchars(strip_tags($_POST['password'])) : 0;
+		$timestamp = time();
 
-		$timestamp = mktime(date('H'),date('i'),0,date('n'),date('j'),date('Y'));
-
-		$entry = nl2br($entry);
-
-		if(!get_magic_quotes_gpc())
```

Comment:
magic quotes was deprecated. It'd automatically add slashes to your title so you could just drop it into a query without worrying about little bobby tables. It was a pretty stupid thing because if you didn't know about it you could do addslashes twice and end up with backslashes all over your database.